lms/static/sass/course/_info.scss

```diff
@@ -56,6 +56,115 @@ div.info-wrapper {
       font-style: normal;
     }
 
+    div.upgrade-banner {
+      // This banner uses the Pattern Library's defined variables
+      @include border-left(0px);
+
+      border: 1px solid $uxpl-gray-background;
+      width: 100%;
+      display: table;
+
+      .notification-color-border {
+        width: 6px; //Value defined by UX team
+        min-height: 100%;
+        margin: 0;
+        display: table-cell;
+        background: $notification-highlight-border-color;
+      }
+
+      .notification-content {
+        display: inline-flex;
+        align-items: center;
+        align-content: flex-start;
+        flex-flow: row wrap;
+        background: $homepage-background;
+        width: 100%;
+        padding: $baseline/2 0;
+        margin-bottom: 0;
+        justify-content: space-between;
+
+        .upgrade-icon {
+          margin: 0;
+          padding: $baseline/2 $baseline;
+          flex-flow: row nowrap;
+          align-items: center;
+          flex: 7 1 50px;
```

Comment:
When I set the flex-grow to 7 the Icon image "grows" large enough to meet the expectations of UX.  This expectation was that the icon would be as tall as the wrapped text and also cause at least 2 words to be wrapped when the notification was at is smallest.